lib/assessment/code_feature_extractor.py

```diff
@@ -464,6 +464,9 @@ def extract_object_and_variable_data(self, node):
               self.features["objects"].append({"identifier": node_info["identifier"], "properties": node_info["value"]["args"], "type": "shape", "start": node_info["start"], "end": node_info["end"]})
               self.features["object_types"]["shapes"] += 1
               self.nodes.append(node)
+            else:
+              self.features["variables"].append({"identifier": node_info["identifier"], "properties": node_info["value"]["args"], "type": "none", "start": node_info["start"], "end": node_info["end"]})
```

Comment:
Added to account for variables that store the output of a function that are not shapes, text, or sprites.